README.md

```diff
@@ -35,23 +37,24 @@ In this project, distribute beta version with [Xcode Cloud](https://developer.ap
 
 ## Contributing
 
-We always welcome any and all contributions! See [CONTRIBUTING.md](CONTRIBUTING.md) for more information
+We always welcome any and all contributions! See [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
 
-For Japanese speakers, please see [CONTRIBUTING.ja.md](CONTRIBUTING.ja.md)
+For Japanese speakers, please see [CONTRIBUTING.ja.md](CONTRIBUTING.ja.md).
 
 ## Requirements
 
-Latest Android Studio **Electric Eel** and higher. You can download it from [this page](https://developer.android.com/studio/preview).
-[iOS Requirements](app-ios/README.md)
+Latest Android Studio **Electric Eel** or higher. You can download it from [this page](https://developer.android.com/studio/preview).
```

Comment:
I fixed because I feel like I often see `or higher`... 🤔
Please tell me if I'm wrong. 🙏🏼 
In that case, I will undo the change.